src/main/java/uk/gov/companieshouse/accounts/association/mapper/AssociationsListCompanyDaoToDtoMapper.java

```diff
@@ -35,15 +36,17 @@ private Function<Association, Association> enrichWithCompanyName( final String c
     protected void enrichWithItems( Page<uk.gov.companieshouse.accounts.association.models.Association> page, @MappingTarget AssociationsList list, @Context Map<String, String> context ) {
         final var companyNumber = context.get( "companyNumber" );
         final var pageContent = page.getContent();
-        final var enrichWithCompanyName = enrichWithCompanyName( companyNumber );
 
-        final var items =
-        pageContent.parallelStream()
-                   .map( associationUserDaoToDtoMapper::daoToDto )
-                   .map( enrichWithCompanyName )
-                   .toList();
+        List<Association> items = List.of();
+        if ( !pageContent.isEmpty() ) {
+            final var enrichWithCompanyName = enrichWithCompanyName(companyNumber);
 
-        list.items( items );
+            items = pageContent.stream()
```

Comment:
you dont need streme. you can map directly.

Reply:
pageContent is a List and so it does not have a map method. It is necessary to stream.